scripts/split-glb-nodes.py

```
@@ -0,0 +1,288 @@
+                raise SystemExit("morph targets are not handled")
+            prim["attributes"] = {k: take_accessor(v) for k, v in prim["attributes"].items()}
+            if "indices" in prim:
+                prim["indices"] = take_accessor(prim["indices"])
+        mesh_map[mi] = len(new_meshes)
+        new_meshes.append(m)
+        return mesh_map[mi]
+
+    out_nodes = []
+    dropped_translations = {}
+
+    def copy_subtree(src_idx, is_root):
```

Ruff (0.16.1)

[warning] Missing return type annotation for private function `copy_subtree`

(ANN202)